App/Redux/ContactsRedux.ts

```diff
@@ -30,7 +30,7 @@ export function reducer (state: ContactsState = initialState, action: ContactsAc
 
 export const ContactsSelectors = {
   isKnown: (state: RootState, id: string) => state.contacts.contacts.some((p) => p.id === id),
-  byThreadId: (state: RootState, id: string) => state.contacts.contacts.filter((contact) => contact.thread_ids.indexOf(id) > -1)
+  byThreadId: (state: RootState, id: string) => state.contacts.contacts.filter((contact) => (contact.thread_ids || []).indexOf(id) > -1)
```

Comment:
`thread_ids` is an optional array in go, so made the corresponding model change in TS... had some ripple effects like this and a few other places in the app's code.